pandas/core/internals.py

```diff
@@ -216,20 +216,25 @@ def make_block(self, values, placement=None, ndim=None):
         if ndim is None:
             ndim = self.ndim
 
-        return make_block(values, placement=placement, ndim=ndim)
+        return make_block(values, placement=placement, ndim=ndim, dtype=dtype)
```

Comment:
maybe remove dtype here? As fastparquet was not using it (and I don't think anybody else will) 
Or otherwise add the same deprecation warning as you did for `make_block_same_class`